### ISSUE TITLE
Feature Request: SoilMod sprayer fillTypes in triggers.lua

### ISSUE THREAD

**Issue by tater99** [Courseplay/courseplay#1146]
Greetings,

It was suggested to me that I should post this here.  I will try to explain as briefly as possible.

If sprayer fill triggers for each SoilMod fill type are added to a map, the CP helper will only use the trigger for "NPK fertilizer" as the trigger fillType for NPK is 'fertilizer'.

I was able to get the CP helper to use all the triggers by adding the SoilMod fill types to triggers.lua in the nonUpdateables section.  Just added new elseif statements for all the fillTypes.  The fillTypes added for SprayerFillTriggers were: fertilizer2, fertilizer3, herbicide, herbicide2, herbicide3, herbicide4, herbicide5, herbicide6, kalk, water.

My request is made on behalf of all map makers who want to make individual sprayer triggers for each SoilMod fillType.  This allows CP helpers to refill with all the new fertilizers and herbicides.

Thank you for consideration of this request.
tater

**Comment by jmj486** [comment]
@tater99 
Will you consider share those line of code?

**Comment by tater99** [comment]
I made some instructions for editing triggers.lua.  Available here:
https://www.dropbox.com/s/f6yailkgidkpte1/Courseplay_Edit_Instructions.pdf?dl=0

The instructions are intended for users of individual SoilMod spray types fill triggers that I have made for map makers.  I can't distribute modified CP scripts, of course, but the instructions will allow end users to make the edits for personal use.  

You also need the development version of SoilMod (2.0.26 or greater) so the new spray types will be registered before the map loads.
